apps/balancer-tools/src/app/poolsimulator/(components)/SwapCurve.tsx

```diff
@@ -317,3 +364,60 @@ export function SwapCurve() {
     />
   );
 }
+
+function getBetaLimitIndexes({
+  analysisTokenOut,
+  analysisTokenIn,
+  tabTokenIn,
+  tabTokenOut,
+  analysisTokenInitialBalance,
+  tabTokenInitialBalance,
+  beta,
+}: {
+  analysisTokenOut: number[];
+  analysisTokenIn: number[];
+  tabTokenIn: number[];
+  tabTokenOut: number[];
+  analysisTokenInitialBalance: number;
+  tabTokenInitialBalance: number;
+  beta: number;
+}) {
+  const analysisTokenSwapAmounts = [...analysisTokenOut]
+    .reverse()
+    .concat(analysisTokenIn);
+
+  const tabTokenSwapAmounts = [...tabTokenIn].reverse().concat(tabTokenOut);
```

Comment:
why concat/reverse? 👀

Reply:
I'll let a comment about this but is just to order all the numbers.